python/smartcrop.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python
+# Author: https://github.com/davidfischer-ch
+# From Repo: https://github.com/smartcrop/smartcrop.py
+
```

Comment:
This file has a ton of config hardcoded in it. Wouldn't you want to be able to fiddle with these values a bit?

Reply:
Good point, we could later extract them if we need to change the smart crop command line.
for now, I suggest to keep it here and see what feedback we can gather from users using it and adapt what we can configure.

Reply:
Good catch about the docs!
I need to update that 👍